<log>
  <summary verdicts='0' failures='0' errors='1'/>
  <section name='My section'>
    <uncaught_exception>
      <verdict_id>My section</verdict_id>
      <class>RuntimeError</class>
      <message>Boo!</message>
      <backtrace>
        <level_0 location='example.rb:9:in `block (2 levels) in test_example&apos;'/>
        <level_1 location='example.rb:8:in `block in test_example&apos;'/>
        <level_2 location='example.rb:7:in `test_example&apos;'/>
      </backtrace>
    </uncaught_exception>
  </section>
</log>
